src/sentry/apidocs/parameters.py

```diff
@@ -109,6 +109,44 @@ class OrganizationParams:
 For example the following are valid parameters:
 - `/?project=1234&project=56789`
 - `/?project=-1`
+""",
+    )
+    OWNER = OpenApiParameter(
+        name="owner",
+        location="query",
+        required=False,
+        type=bool,
+        description="""Specify `true` to restricts results to organizations in which you are an owner.""",
+    )
+    QUERY = OpenApiParameter(
+        name="query",
+        location="query",
+        required=False,
+        type=str,
+        description="""The search filter for your query, read more about query syntax [here](https://docs.sentry.io/product/sentry-basics/search/).
```

Comment:
```suggestion
        description="""Filters results by using [query syntax](https://docs.sentry.io/product/sentry-basics/search/).
```

Reply:
should i update the other endpoint docs that use this language?

Reply:
Yeah that's a good idea, we should try to consolidate text for similar endpoints

Reply:
i'm gonna update the other param that uses this